changelog.md

```diff
@@ -12,6 +12,9 @@
 
 - Specific warnings can now be turned into errors via `--warningAsError[X]:on|off`.
 - The `define` and `undef` pragmas have been de-deprecated.
+- JavaScript backend adds compile-time optional `let` variable declarations,
+  compile Nim with `-d:nimJsVar` for `var` and without for `let`,
```

Comment:
instead how about using the standard flag `--legacy` which is exactly for this purpose;
eg `--legacy:allowSemcheckedAstModification|checkUnsignedConversions`

=>
`--legacy:jsusevar` or `--legacy:jsvar`